### DISCUSSION TITLE
SQL Editor: Smart mode for 'Blank line as statement delimiter' option was added

### DISCUSSION THREAD

**Discussion by hlapidez** [dbeaver/dbeaver#29267] [Q&A] [answered]
In community version 24.0.3
https://dbeaver.io/2024/04/21/dbeaver-24-0-3/

Hello, 

I am searching for the option too turn this feature on and off, but I am not able to find it in the preferences dialog. 

Thanks for a hint

**Comment by E1izabeth** [comment, marked as answer]
Here it is
![image](https://github.com/dbeaver/dbeaver/assets/28875055/142eb0b5-f1d7-4003-80f6-2c8a5f7efc83)

**Comment by E1izabeth** [comment]
We encountered a bug connected with this new mode. The fix will be included in DBeaver 24.0.4.
Also you can try Early Access version with fix https://dbeaver.io/files/ea/ now.
